extensions/smallrye-graphql/runtime/src/main/java/io/quarkus/smallrye/graphql/runtime/SmallRyeGraphQLRuntimeConfig.java

```diff
@@ -13,4 +13,12 @@ public class SmallRyeGraphQLRuntimeConfig {
     @ConfigItem(name = "ui.enable", defaultValue = "true")
     boolean enable;
 
+    /**
+     * Specifies the field visibility for the GraphQL schema.
+     * This configuration item allows you to define a list of patterns (Class.field.
```

Comment:
- missing closing parenthesis here
- I think we should mention the special value `no-introspection` and maybe also add a link to the graphql-java docs that talk about this?